jax/_src/dispatch.py

```diff
@@ -814,11 +820,12 @@ def xla_executable(self):
 
   @staticmethod
   def from_trivial_jaxpr(jaxpr, consts, device, in_avals, out_avals, effects,
-                         kept_var_idx) -> XlaCompiledComputation:
+      kept_var_idx, keepalive: Optional[Any]) -> XlaCompiledComputation:
```

Comment:
```suggestion
      kept_var_idx, keepalive: Optional[Any]) -> XlaCompiledComputation:
    assert keepalive is None
```
From what I can tell, we're accepting an optional `keepalive` here that will always be `None`, only in order to accommodate having all the compilation arguments ("`compile_args`") passed around in bulk in other parts in this file. If what I'm saying sounds right, what do you think about documenting this by adding this assertion?

Reply:
Yep that sounds right, adding the assertion.